src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main implements Fillable {
+
+    public static void main(String[] args) {
+        Main main = new Main();
+        Shape[] shapeArr = main.fill();
+
+        for (Shape s : shapeArr) {
+            System.out.println(s.info());
+        }
+    }
+
+    public Shape[] fill() {
+        Random rand = new Random();
+        Shape[] shapeArr = new Shape[generateRandomArrayLength()];
+        Shape shape = null;
+        for (int i = 0; i < shapeArr.length; i++) {
```

Comment:
this logic better to have in the separate class

Reply:
done!